Snowflake.Data/Core/Session/SFSession.cs

```diff
@@ -94,7 +94,7 @@ internal void ProcessLoginResponse(LoginResponse authnResponse)
                     logger.Debug("Query context cache disabled.");
                 }
                 logger.Debug($"Session opened: {sessionId}");
-                _startTime = DateTimeOffset.UtcNow.ToUnixTimeSeconds();
+                _startTime = DateTimeOffset.UtcNow.ToUnixTimeMilliseconds();

```

Comment:
check @sfc-gh-mhofman  if this isn't used with [s] still in mind

Reply:
yes, I did